File: test/bench/fiber/fiber.php
<?php
$count = 0;

$fibers = array();

for ($i = 0; $i < 100000; $i++) {
    $f = new Fiber(function () use (&$count) {
        $count += 1;
        Fiber::suspend();
        $count += 1;
    });
    $f->start();
    array_push($fibers, $f);
}

foreach ($fibers as $f) {
    $f->resume();
}
echo $count;
?>

File: src/test/bench/fiber/fiber.rb
$count = 0

fibers = []
100000.times do |i| 
    f = Fiber.new do 
        $count += 1
        Fiber.yield
        $count += 1
    end
    f.resume()
    fibers << f
end

fibers.each {|f| f.resume() }

puts $count